code/Model/Submission/SubmittedFileField.php

```diff
@@ -16,68 +17,89 @@
  * @package userforms
  * @property int $UploadedFileID
  * @method File UploadedFile()
+ * @method ManyManyList<File> UploadedFiles()
  */
 class SubmittedFileField extends SubmittedFormField
 {
     private static $has_one = [
         'UploadedFile' => File::class
     ];
 
+    private static $many_many = [
+        'UploadedFiles' => File::class
+    ];
```

Comment:
Feels bad having both of these, but I guess we have to do this until the next major release.